builtin/logical/aws/iam_policies.go

```diff
@@ -0,0 +1,102 @@
+package aws
+
+import (
+	"context"
+	"encoding/json"
+	"net/url"
+
+	"github.com/aws/aws-sdk-go/aws"
+	"github.com/aws/aws-sdk-go/service/iam"
+	"github.com/aws/aws-sdk-go/service/iam/iamiface"
+	"github.com/hashicorp/vault/sdk/logical"
+)
+
+// PolicyDocument represents an IAM policy document
+type PolicyDocument struct {
+	Version   string
+	Statement []StatementEntry
```

Comment:
Could we rename this to `Statements` since it can have many of them?

Reply:
Yep, good call, changed in e1a925a65.